apps/application-system/api/src/app/modules/application/application.controller.ts

```diff
@@ -1128,6 +1139,48 @@ export class ApplicationController {
         'Users role does not have permission to delete this application in this state',
       )
     }
+
+    // delete charge in FJS
+    this.deleteApplicationCharge(existingApplication)
```

Comment:
this could fail and we would still delete the payment and application on our side.